contrib/platform/test/com/sun/jna/platform/win32/VersionUtilTest.java

```diff
@@ -0,0 +1,37 @@
+/* This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import java.io.File;
+
+import junit.framework.TestCase;
+
+public class VersionUtilTest extends TestCase {
+
+    public static void main(String[] args) {
+        junit.textui.TestRunner.run(VersionUtilTest.class);
+    }
+
+    public void testGetFileVersionNumbers() {
+        File file = new File(System.getenv("SystemRoot"), "regedit.exe");
+        assertTrue(file.exists());
+
+        int[] version = VersionUtil.getFileVersionNumbers(file.getAbsolutePath());
+        assertNotNull("Version info should have been returned.", version);
+        assertTrue("Four fields should have been returned.", version.length == 4);
+
+        assertTrue("No field should less than or equal to 0 when pulling version from regedit.exe", version[0] > 0);
+        assertTrue("No field should less than or equal to 0 when pulling version from regedit.exe", version[1] > 0);
+        assertTrue("No field should less than or equal to 0 when pulling version from regedit.exe", version[2] > 0);
+        assertTrue("No field should less than or equal to 0 when pulling version from regedit.exe", version[3] > 0);
```

Comment:
Don't use the same comment in order to be able to decide which assertion failed:
```java
assertTrue("Major should not be <= 0", version[0] > 0);
assertTrue("Minor should not be <= 0", version[0] > 0);
...etc...
```